perceval/runtime/_token_management.py

```diff
@@ -46,61 +43,64 @@ class TokenProvider:
     - File on the disk
     """
 
-    _CACHED_TOKEN = None
-
     def __init__(self, env_var: str = "PCVL_CLOUD_TOKEN"):
```

Comment:
I would say that the __init__ of the TokenProvider should be marked "deprecated", not its methods.
Nobody uses it anymore in our code, I guess. Or are there still a usage somewhere?